tsl/test/sql/continuous_aggs_permissions.sql

```diff
@@ -97,7 +97,7 @@ REVOKE EXECUTE ON FUNCTION get_constant() FROM PUBLIC;
 
 \c  :TEST_DBNAME :ROLE_DEFAULT_PERM_USER_2
 \set ON_ERROR_STOP 0
-select from alter_job_schedule(:cagg_job_id, max_runtime => NULL);
+select from alter_job(:cagg_job_id, max_runtime => NULL);
```

Comment:
Nit: I'm surprised that this doesn't give a syntax error, but it does look strange.

Reply:
targetlist can be empty